README.md

```diff
@@ -60,7 +60,9 @@ This will install `http-server` globally so that it may be run from the command
 
 `--cors` Enable CORS via the `Access-Control-Allow-Origin` header
 
-`-o` Open browser window after staring the server
+`-o` Open browser window after starting the server
+
+`-u` Url to display in the opened browser (in case you have an alias to your localhost)
```

Comment:
could we actually change this to `--url` to avoid confusion with `--utc` and `--username`?

Reply:
No, it's not possible because `-o` doesn't let you specify the desired url to open. I'll check the conflicts (3 years after!).